src/instructlab/config/init.py

```diff
@@ -410,5 +424,16 @@ def get_params(
         model_path = utils.expand_path(
             click.prompt("Path to your model", default=model_path)
         )
+        click.echo()
 
     return model_path, taxonomy_path, cfg
+
+
+def get_separator(text: str) -> str:
+    text_length = len(text)
+    try:
+        terminal_width = shutil.get_terminal_size().columns
+    except Exception:  # pylint: disable=broad-exception-caught
```

Comment:
Do we know the cases this exception is happening?  Is there a particular type of exception that's being thrown?  A comment could be helpful here to explain.

Reply:
Actually, I tried to test in `Linux console` or `MacOS terminal` , so far cannot reproduce the error, tried to search, seems might hit `OSError`

Reply:
It does look like it would be an OSError.

Maybe just add a comment then explaining why the exception is being caught => Exception can occur when running non interactive scenarios where a terminal isn't associated with the process